tests/baselines/reference/user/axios-src.log

```diff
@@ -1,5 +1,7 @@
 Exit Code: 1
 Standard output:
+../../../../../built/local/lib.es2015.iterable.d.ts(41,6): error TS2300: Duplicate identifier 'IteratorResult'.
+../../../../../node_modules/@types/node/index.d.ts(77,11): error TS2300: Duplicate identifier 'IteratorResult'.
```

Comment:
I think this is a result of depending on old versions of `@types/node`. dtslint-runner had this problem until I upgraded from a two-year-old version.